qa/tasks/notification_tests.py

```diff
@@ -186,31 +232,47 @@ def run_tests(ctx, config):
     for client, client_config in config.items():
         (remote,) = ctx.cluster.only(client).remotes.keys()
 
+        attr = ["!kafka_test", "!amqp_test"]
```

Comment:
why do you need this?
why not running all tests?

Reply:
This will be needed because kafka tests would fail if we run them along with AMQP tests and the reason is the setup of those are in different yaml files. Vice-versa will also go the same way. Hence I've added these. And if we don't add any `extra_attr` in the yaml file then the remaining tests will be executed.

Reply:
ok. i see, but you are not running any of the other tests in teuthology.
according to the above you should execute the suite 3 times:
* kafka attr
* amqp attr
* no attr

Reply:
This is fixed and now it's running all the tests in teuthology: https://pulpito.ceph.com/kapandya-2021-06-01_06:57:20-rgw:notifications-wip-rgw-rabbitmq-distro-basic-smithi/